docs/manual/interpreters.md

```diff
@@ -0,0 +1,39 @@
+---
+layout: page
+title: "Interpreters"
+description: ""
+group: manual
+---
+{% include JB/setup %}
+
+
+## Interpreters in zeppelin
+
+This section explain the role of Interpreters, interpreters group and interpreters settings in Zeppelin.
+Zeppelin interpreter concept allows any language/data-processing-backend to be plugged into Zeppelin.
+Currently Zeppelin support many interpreters such as Scala(with Apache Spark), Python(with Apache Spark), SparkSQL, Hive, Markdown and Shell.
+
+### What is zeppelin interpreter?
+
+Zeppelin Interpreter is the plug-in which enable zeppelin user to use a specific language backend. For example to use scala code in Zeppelin, you need scala interpreter.
```

Comment:
Is this confusing? there isn't a "Scala" interpreter - there is only a Spark interpreter on the list